bsd/models.py

```diff
@@ -385,6 +386,22 @@ def _get_absolute_url(self):
         return get_bsd_event_url(self.event_id_obfuscated)
     absolute_url = property(_get_absolute_url)
 
+    def _get_start_datetime_utc(self):
+        event_time_zone = pytz.timezone(self.start_time_zone)
+        naive_datetime = datetime.datetime.strptime(
+            str(self.start_day) + ' ' + str(self.start_time),
+            '%Y-%m-%d %H:%M:%S'
+        )
+        local_datetime = event_time_zone.localize(naive_datetime)
+        utc_datetime = local_datetime.astimezone(pytz.utc)
+        return utc_datetime
+    start_datetime_utc = property(_get_start_datetime_utc)
+
+    def _is_approved(self):
+        if not self.flag_approval == 1:
+            return True
```

Comment:
`return not self.flag_approval == 1`?

Reply:
Also wonder if we can just remove/replace the other `status` property now that we have this? I like this new implementation, although now the other thing seems redundant.